main.cc

```diff
@@ -967,13 +967,13 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
 
             supervisor::notify("creating and verifying directories");
             utils::directories::set dir_set;
-            dir_set.add(cfg->data_file_directories());
-            dir_set.add(cfg->commitlog_directory());
-            dir_set.add(cfg->schema_commitlog_directory());
+            dir_set.add(dirs->get_data_file_dirs());
+            dir_set.add(dirs->get_commitlog_dir());
+            dir_set.add(dirs->get_schema_commitlog_dir());
```

Comment:
You remove this place in the next patch, so what's the point?

Reply:
Fixed - this patch has been merged with the next one. Thanks!